src/sentry/integrations/slack/message_builder/notifications/rule_save_edit.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import annotations
+
+from sentry.integrations.slack.message_builder import SlackBlock
+from sentry.integrations.slack.message_builder.base.block import BlockSlackMessageBuilder
+from sentry.integrations.slack.utils.escape import escape_slack_text
+from sentry.models.rule import Rule
+from sentry.notifications.types import NotificationSettingEnum
+from sentry.utils.http import absolute_uri
+
+
+class SlackRuleSaveEditMessageBuilder(BlockSlackMessageBuilder):
+    def __init__(
+        self,
+        rule: Rule,
+        new: bool,
+    ) -> None:
+        super().__init__()
```

Comment:
any other kwargs to pass through to super init?

Reply:
Doesn't seem like it